crud.go

```diff
@@ -51,8 +54,8 @@ type lolrusDoc struct {
 
 // Creates a simple in-memory Bucket, suitable only for amusement purposes & testing.
 // The Bucket is created empty. There is no way to save it persistently.
-func NewBucket(bucketName string) Bucket {
-	ohai("NewBucket %s", bucketName)
+func NewBucket(bucketName string) sgbucket.Bucket {
+	log.Printf("NewBucket %s", bucketName)
```

Comment:
It looks like we're losing the conditional logging and "Walrus:" prefixing from ohai - I think we'd want to keep both of those.